packages/cli/src/plugins/resource/plugin-node-modules.js

```diff
@@ -142,6 +142,8 @@ const walkPackageJson = (packageJson = {}) => {
             packageExport = esmPath
               ? esmPath
               : fallbackPath;
+          } else if (exportMapEntry.default) {
```

Comment:
looks like I overlooked something, may need to extract this out, depending on how long the TODOs take.